packages/amp/src/__tests__/amp.test.ts

```diff
@@ -0,0 +1,193 @@
+import { createStore, InitializedStore } from "@frontity/connect";
+import clone from "clone-deep";
+import wpSource from "@frontity/wp-source/src";
+import WpSource from "@frontity/wp-source/types";
+import AMP from "../../types";
+import { MergePackages } from "frontity/types";
+import Amp from "..";
+
+let store: InitializedStore<MergePackages<WpSource, AMP>>;
+describe("AMP tests", () => {
+  beforeEach(() => {
+    // Initialize the store
+    store = createStore<MergePackages<WpSource, AMP>>(
```

Comment:
You can use the `Packages` type here.

```suggestion
import AMP, { Packages } from "../../types";
import Amp from "..";

let store: InitializedStore<Packages>;
describe("AMP tests", () => {
  beforeEach(() => {
    // Initialize the store
    store = createStore<Packages>(
```